### PR TITLE
installation: Update password check for live CD issue, old soft failure, remove sleep

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

**Comment by okurz** [comment]
@asmorodskyi

File: tests/installation/user_settings.pm [hunk 1]
```diff
@@ -43,13 +41,17 @@ sub run() {
...
     if (check_screen('inst-userpasswdtoosimple', 13)) {
         send_key "ret";
     }
     else {
-        record_soft_failure 'bsc#937012';
+        if (check_var('LIVECD', 1)) {
+            record_soft_failure 'boo#1013206';
+        }
+        else {
+            record_soft_failure 'bsc#937012';
```
Comment (asmorodskyi) [inline tests/installation/user_settings.pm]: We need to die here , not soft fail on closed bug

Comment (coolo) [inline tests/installation/user_settings.pm]: In maintenance testing we also install old products. After you checked they are not affected, I'm fine with a die

Comment (coolo) [inline tests/installation/user_settings.pm]: bingo -> https://openqa.suse.de/tests/648953#step/user_settings/6

Comment (coolo) [inline tests/installation/user_settings.pm]: so skip this on check_var('VERSION', '12') && check_var('ARCH', 's390x') and assert everywhere else.

Comment (okurz) [inline tests/installation/user_settings.pm]: done

**Comment by okurz** [comment]
wait, I need to check the user_settings_root

**Comment by okurz** [comment]
Updated:
* Extracted common methods for `user_settings` and `user_settings_root` with a new approach, multiple inheritance to include a base class in both cases

Local verification run: http://lord.arch/tests/5432

**Pull request merged**